runtime/compiler/build/rules/gnu/common.mk

```diff
@@ -65,6 +69,11 @@ JIT_DIR_LIST+=$(dir $(JIT_PRODUCT_BUILDNAME_SRC))
 jit_cleanobjs::
 	rm -f $(JIT_PRODUCT_BUILDNAME_SRC)
 
+ifneq ($(JITSERVER_SUPPORT),)
+    protoc: $(PROTO_GEN_DIR)/compile.pb.h
+        $(call RULE.proto,$(PROTO_GEN_DIR)/compile,$(PROTO_DIR)/compile.proto)
```

Comment:
Because `protoc` is `.PHONY` this rule will execute in every build.

The call to `RULE.proto` should not be indented more than the line before. As is, it suggests it is part of the script for the `protoc` rule which it is not (because it doesn't begin with a tab).

Reply:
Good to know. Thanks! Will decrease the indentation.